### PR TITLE
Feature product categories

### PR DESCRIPTION
**Pull request by fishbacon**
Working on #78.

- [x] Create category  
- [x] Merge commit because I branched from my local next
- [x] Add categories to admin page
- [x] Change multiple migrations to single migration for pull request
- [x] Add to ranking pages
- [x] Tests!

Nice to have features, possible enhancements for after merge:
- [ ] Link to the category from products listing
- [ ] Instantly add products to new category (product listing on edit category page)
- [ ] Add fields for start and end time of the ranking

### CODE REVIEW

**Comment by fishbacon** [comment]
> Maybe this isn't super important since we're really in going to use it in the admin interface, but just getting it out of the code is worth it to me.

Down the line categories could do more but I agree that the feature is currently quality of life for our volunteers.

I think using this feature to improve stregreport would be great.

**Comment by fishbacon** [comment]
With this commit we kinda fulfil the whole issue and I am probably going to recommend merging now.

I will be creating issues for the rest of the things in the list in the top comment.

File: treo/settings.py [hunk 1]
```diff
@@ -183,7 +183,7 @@
 
 USE_L10N = True
 
-USE_TZ = True
+USE_TZ = False
```
Comment (DelusionalLogic) [inline treo/settings.py]: I'm not sure what that's doing in here.

Comment (fishbacon) [reply treo/settings.py]: Dunno, probably slipped up. This was to avoid the errors related to #84. Will revert it.

**Comment by josmad** [comment]
I cleaned up and squashed the commits and rebased it on top of next in preparation to merge

**Pull request merged**